testnets/qubeticstestnet/assetlist.json

```diff
@@ -0,0 +1,32 @@
+{
+  "$schema": "../../assetlist.schema.json",
+  "chain_name": "qubeticstestnet",
+  "assets": [
+    {
+      "description": "The native EVM, governance and staking token of the Qubetics Chain",
+      "denom_units": [
+        {
+          "denom": "tics",
+          "exponent": 0
+        },
+        {
+          "denom": "tics",
```

Comment:
Cannot have two different units with the same identifier (denom). Please rename to something else (and also update 'display' unit accordingly